birdset/modules/models/linear_classifier.py

```diff
@@ -23,4 +23,4 @@ def __init__(
     def forward(
         self, input_values: torch.Tensor, labels: Optional[torch.Tensor] = None
     ) -> torch.Tensor:
-        return self.classifier(input_values) 
\ No newline at end of file
+        return self.classifier(input_values).squeeze(1) 
```

Comment:
Why `squeeze(1)` here? Do you have 2-dimensional embeddings? If so, I would suggest putting the squeezing in the “get embeddings” function of the model so that all embeddings are 1 dimensional.

Reply:
No the embeddings are saved 1-D but I think through the pipeline a dimension with size 1 is added 🤔